mathesar_ui/src/stores/table-data/meta.ts

```diff
@@ -211,16 +211,35 @@ export class Meta {
           );
         }
         if ($filter.filters?.length > 0) {
-          const filter = {};
+          // Specify fields to pass style checks for special handling of get_duplicates op
+          const filter: {
+            and: {
+              field: string,
+              op: string,
+              value: string
+            }[]
+          } = { and: [] };
+
           const terms = $filter.filters.map((term) => ({
             field: term.column.id,
             op: term.condition.id,
             value: term.value,
           }));
           filter[$filter.combination.id as string] = terms;
-          params.push(
-            `filters=${encodeURIComponent(JSON.stringify(filter))}`,
-          );
+          if ('and' in filter && filter.and.length === 1 && filter.and[0].op === 'get_duplicates') {
+            // Special handling for the get_duplicates operation due to backend bug
+            // The backend can only handle the get_duplicates operation by itself
+            // The "and" must be removed and the value must be in an array
+            filter.and[0].value = [filter.and[0].field];
```

Comment:
@yankevn This line seems to be causing the issue. `value` is of type `string` and not an `array of string`. @seancolsen Can you look into this?

Reply:
That makes sense. The backend currently requires that the value field is an `array of string` rather than `string`, which is why that is there. If I remove the brackets, the API call fails.

Reply:
Can the [`value` field](https://github.com/centerofci/mathesar/pull/844/files/653eef286526205f5aa791f1bf673a8001d442c2#diff-7bb52cec499a124f38598455ace9ea0fd1bde7e2861a44f3052d5dd94b9b40efR219) be an `array of string`?

Reply:
I didn't expect that to work, but it actually works just fine with our existing filter implementation. I'll push that right now!

Reply:
This section is hacky but I don't think we can do anything about it until we resolve https://github.com/centerofci/mathesar/issues/846. So, I'm proceeding with merging these changes as is.